src/sql/src/func.rs

```diff
@@ -1820,16 +1820,34 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "ascii" => Scalar {
             params!(String) => UnaryFunc::Ascii(func::Ascii) => Int32, 1620;
         },
+        // Note: this is the original version of the AVG(...) function, as it existed prior to
+        // v0.66. We updated the internal type promotion used when summing values to increase
+        // precision, but objects (e.g. materialized views) that already used the AVG(...) function
+        // could not be changed. So we migrated all existing uses of the AVG(...) function to this
+        // version.
+        //
+        // TODO(parkmycar): When objects no longer depend on this function we can safely delete it.
+        "avg_internal_v1" => Scalar {
```

Comment:
Sorry about that, I forgot to squash the later commit down, should be updated now! The `test_compare_builtins_postgres` failed exactly like you thought it would, which is what resulted in my changing the oids.

Also I made the change to put `avg_internal_v1` into `mz_catalog`